src/components/todoitem.js

```diff
@@ -37,10 +39,7 @@ export default class TodoItem {
     }
     check.addEventListener('change', (e) => {
       this.completed = !this.completed;
-      //* if checkbox is checked input is disabled
-      if (e.target.checked) input.setAttribute('disabled', 'true');
-      //* if checkbox is not checked input is enabled
-      else if (!e.target.checked) input.removeAttribute('disabled');
+      checkHandler(e, input);
```

Comment:
- [ ] Upon marking a given todo item as _completed_ , the cahnges are not being reflected in local storage hence on page reloading, the changes disappear, would you fix that issue?
- [ ] The same issue happens when we edit a given todo item, kindly watch the following screencast for more context


https://user-images.githubusercontent.com/56886064/149422443-e2a14af1-138b-470f-b84f-356e2d546d8d.mp4

Reply:
Thank you for the video 👍